pkg/ddl/ingest/backend.go

```diff
@@ -53,16 +52,6 @@ type BackendCtx interface {
 	// BackendCtx.
 	Register(indexIDs []int64, uniques []bool, tableName string) ([]Engine, error)
 	UnregisterEngines()
-	// FinishImport imports all Register-ed engines of into the storage, collects
-	// the duplicate errors for unique engines.
-	//
-	// TODO(lance6716): unify with CollectRemoteDuplicateRows.
```

Comment:
change this TODO to "remove the indexID argument from CollectRemoteDuplicateRows"

Reply:
See https://github.com/pingcap/tidb/pull/54149/commits/545c9e9c4bd59aa756d688ab4dae9cfdd9248d9d